doc/schema/schema.md

```diff
@@ -66,18 +66,28 @@ Moreover, each group contains the following attributes:
 Samples from the posterior distribution p(theta|y).
 
 ### `sample_stats`
-Information and diagnostics for each `posterior` sample, provided by the inference backend. It may vary depending on the algorithm used by the backend (i.e. an affine invariant sampler has no energy associated). The name convention used for `sample_stats` variables is the following:
-* `lp`: (unnormalized) log probability for sample
-* `step_size`
-* `step_size_bar`
-* `tune`: boolean variable indicating if the sampler is tuning or sampling
-* `depth`:
-* `tree_size`:
-* `mean_tree_accept`:
-* `diverging`: HMC-NUTS only, boolean variable indicating divergent transitions
-* `energy`: HMC-NUTS only
-* `energy_error`
-* `max_energy_error`
+Information and diagnostics for each `posterior` sample, provided by the inference
+backend. It may vary depending on the algorithm used by the backend (i.e. an affine
+invariant sampler has no energy associated). Therefore none of this parameters
+should be assumed to be present in the `sample_stats` group. The convention
+below serves to ensure that _if_ a variable is present with one of these names
+it will correspond to the definition included here.
+
+The name convention used for `sample_stats` variables is the following:
+
+* `lp`: The log posterior density (up to an additive constant).
+* `acceptance_rate`: The average acceptance probabilities of all possible samples in the proposed tree.
+* `step_size`: The current integration step size.
+* `step_size_nom`: The nominal integration step size. The `step_size` may differ from this, for example if the step size is jittered. Should only be present if `step_size` is also present and it varies between samples (i.e. step size is jittered).
+* `tree_depth`: The number of tree doublings in the balanced binary tree.
+* `n_steps`: The number of leapfrog steps computed.
+* `diverging`: (boolean) Indicates the presence of leapfrog transitions with large energy deviation from starting and subsequent termination of the trajectory.
+* `energy`: The value of the Hamiltonian energy for the accepted proposal (up to an
+additive constant).
```

Comment:
I think so far this is correct and is true for all libraries, however I have no idea about what should it say (if anything) about what is `energy` when there is a diverging transition. Not sure if this propagates to `energy_error` or if defining what does `accepted proposal` means when diverging is enough.

Reply:
I had thought that if a transition was divergent then the entire transition was rejected, i.e. the returned sample was always the starting point. In an earlier comment I think @ahartikainen said that's not what Stan at least does. I've been meaning to look into this.

Reply:
Yeah @ahartikainen was totally right about this. When a divergent transition is encountered, the tree doubling is rejected, but the already-accepted tree is not.